scripts/package-release-assets.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+mkdir release
+cd artifacts || exit
+
+for dir in ouch-*.exe; do
+    mv "$dir" "${dir%.exe}"
+done
+
+for target in ouch-*; do
+    [ -f "$target/ouch" ] && chmod +x "$target/ouch"
+    cp -r artifacts "$target/completions"
+    mkdir "$target/man"
+    mv "$target"/completions/*.1 "$target/man"
+    cp ../{README.md,LICENSE,CHANGELOG.md} "$target"
+    tar czf "../release/$target.tar.gz" "$target"
```

Comment:
If `"windows"` in `"target"`, perhaps we want to use `zip` instead of `tar.gz`.

Similar to what it's done in [bat](https://github.com/sharkdp/bat/releases) and [ripgrep](https://github.com/BurntSushi/ripgrep/releases).